ush/forecast_postdet.sh

```diff
@@ -545,22 +545,23 @@ CICE_postdet() {
 
   # Link CICE forecast output files from DATA/CICE_OUTPUT to COM
   local source_file dest_file
-  for fhr in ${CICE_OUTPUT_FH}; do
-    fhr3=$(printf %03i "${fhr}")
+  for fhr in "${CICE_OUTPUT_FH[@]}"; do
 
+    fhr=$((10#${fhr}))
```

Comment:
This shouldn't be necessary, since CICE_OUTPUT_FH should already lack leading zeros.

Reply:
@WalterKolczynski-NOAA When I remove this line, there is an error using printf.
`/lfs/h2/emc/ens/noscrub/eric.sinsky/GEFS/gw_rfcst_prod2/ush/forecast_postdet.sh: line 556: printf: 120
: invalid number`
I think when mapfile is used to generate the forecast hour list, the list does not contain base10 values.

Reply:
Looks like the last value in the sequence gets an erroneous new-line at the end:
```
$ export FHMIN=0
$ export FHOUT_ICE=24
$ export FHMAX=120
$ mapfile -t -d ' ' CICE_OUTPUT_FH < <(seq -s ' ' "${FHMIN}" "${FHOUT_ICE}" "${FHMAX}")
$ for fhr in "${CICE_OUTPUT_FH[@]}"; do echo "==>${fhr}<=="; done
==>0<==
==>24<==
==>48<==
==>72<==
==>96<==
==>120
<==
```